keras_cv/layers/preprocessing/random_brightness.py

```diff
@@ -0,0 +1,119 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+
+_SCALE_VALIDATION_ERROR = ('The `scale` should be number or a list of two numbers '
+                           'that ranged between [-1.0, 1.0]. ')
+
+
+class RandomBrightness(tf.keras.layers.Layer):
+  """Randomly adjust brightness for the a RGB image.
+
+  This layer will randomly increase/reduce the brightness for the input RGB image.
+  During inference time, the output will be identical to input. Call the layer with
+  training=True to adjust brightness of the input.
+
+  Note that same brightness adjustment will be apply to all the images in the same
+  batch.
+
+  Args:
+    scale: Float or a list/tuple of 2 floats between -1.0 and 1.0. The scale is
+      used to determine the lower bound and upper bound of the brightness adjustment.
+      A float value will be choose randomly between the limits. When -1 is chosen,
+      the output image will be black, and when 1.0 is chosen, the image will be
+      fully white. When only one float is provided, eg, 0.2, then -0.2 will be used
+      for lower bound and 0.2 will be used for upper bound.
+    seed: integer, for fixed RNG behavior.
+
+  Inputs:
+    3D (HWC) or 4D (NHWC) tensor, with float or int dtype. The value should be ranged
+    between [0, 255].
+
+  Output:
+    3D (HWC) or 4D (NHWC) tensor with brightness adjusted based on the `scale`. The
+    output will have same dtypes as the input image.
+
+  Sample usage:
+  ```
+    random_bright = keras_cv.layers.RandomBrightness(scale=0.2)
+    # An image with shape [2, 2, 3]
+    image = [[[1, 2, 3], [4 ,5 ,6]],
+             [[7, 8, 9], [10, 11, 12]]]
+    # Assume we randomly select the scale to be 0.1, then it will apply 0.1 * 255 to
+    # all the channel
+    output = random_bright(image, training=True)
+    # output will be int64 with 25.5 added to each channel and round down.
+    tf.Tensor(
+      [[[26 27 28]
+        [29 30 31]]
+       [[32 33 34]
+        [35 36 37]]], shape=(2, 2, 3), dtype=int64)
+  ```
+  """
+
+  def __init__(self, scale, seed=None, **kwargs):
+    super().__init__(**kwargs)
+    self._set_scale(scale)
+    self._seed = seed
+
+  def _set_scale(self, scale):
+    if isinstance(scale, (tuple, list)):
+      if len(scale) != 2:
+        raise ValueError(_SCALE_VALIDATION_ERROR +  f'Got {scale}')
+      self._check_scale_range(scale[0])
+      self._check_scale_range(scale[1])
+      self._scale = sorted(scale)
+    elif isinstance(scale, (int, float)):
+      self._check_scale_range(scale)
+      scale = abs(scale)
+      self._scale = [-scale, scale]
+    else:
+      raise ValueError(_SCALE_VALIDATION_ERROR +  f'Got {scale}')
+
+  @staticmethod
+  def _check_scale_range(input_number):
+    if input_number > 1.0 or input_number < -1.0:
+      raise ValueError(_SCALE_VALIDATION_ERROR +  f'Got {input_number}')
+
+  def call(self, image, training=None):
```

Comment:
Are we handling a single image or images/batch?

Reply:
We handle both single image and batch image. See the docstring and unit test.

Reply:
Yes I know but I meant that we could standardize the names in our API.
Sometimes we use image in other ops images when we handle batches.

Reply:
I think `inputs` would be appropriate. This follows what every other keras layer calls it.

Reply:
Done. changed to  `inputs` for consistency.

Reply:
Ok but, also if minor, we have the same issue in the next internal/private method. I think we need to make some final choice.